protocol/src/main/java/org/fido/iot/protocol/CryptoService.java

```diff
@@ -677,7 +678,17 @@ public boolean verify(PublicKey verificationKey,
       }
       return false;
     }
-    final Composite header1 = cose.getAsComposite(Const.COSE_SIGN1_PROTECTED);
+
+    final Composite header1;
+    final Object rawHeader = cose.get(Const.COSE_SIGN1_PROTECTED);
+    if (rawHeader instanceof byte[]) {
+      final byte[] protectedHeader = cose.getAsBytes(Const.COSE_SIGN1_PROTECTED);
+      header1 = Composite.fromObject(protectedHeader);
+    } else if (rawHeader instanceof Map) {
```

Comment:
Question: Could this be a map? As per my understanding, it'd only be either a bstr of size 0 or map wrapped in bstr

Reply:
A valid point.  The 'map' clause was leftover from testing.  I will upload a patch for this ASAP.